### ISSUE TITLE
uncaught error in getDynamicIframeWindow

### ISSUE THREAD

**Issue by Thorin-Oakenpants** [arkenfox/TZP#76]
FYI @abrahamjuliot 

Testing with [BP Block Font Fingerprint](https://addons.mozilla.org/firefox/addon/bp-block-font-fingerprint/) , the name in about:addons is `BP Privacy Bock All Font and Glyph Detection` for some reason, which is garbage and three years old, and doesn't alter the glyph results, but does error the font tests ... I get the same error in `getDynamicIframeWindow` as I do in `getFonts`

getFonts - catches the error
https://github.com/arkenfox/TZP/blob/9bd09c342f28de35c732b22f1b7cc23239adc030/js/fonts.js#L313-L317

`getDynamicIframeWindow` doesn't, code starts here
https://github.com/arkenfox/TZP/blob/9bd09c342f28de35c732b22f1b7cc23239adc030/js/generic.js#L177

![uncaught](https://user-images.githubusercontent.com/16656956/118420551-a2649e80-b6ae-11eb-99f7-bb7bab0a2dc5.png)

see https://github.com/arkenfox/TZP/commit/506b313be1550a0ed7b20368f1d13d5947b50eb9

**Comment by Thorin-Oakenpants** [comment]
So it seems in this case it was just nested iframes
![nested](https://user-images.githubusercontent.com/16656956/118427280-5ddfff80-b6bc-11eb-9633-8277481e8e4d.png)

I still need to clean up and refactor my iframe ua returns handle everything properly

**Comment by abrahamjuliot** [comment]
I used to have a chrome extension by BrowserPlugs that had a lot of settings and a nice clean material android-like interface, but functionality and code were very buggy. Then, some time ago, it was removed from the extension store for bad code or something. I got a chrome security warning to remove the extension. There are still a few smaller BrowserPlugs extension/addons but the dev's website has been offline for months. It now redirects to what appears to be malware.
